nav2_costmap_2d/include/nav2_costmap_2d/denoise/image.hpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2022 Andrey Ryzhikov
```

Comment:
Please correct to the year 2023 here and in all files

Reply:
Fixed 1901f819228342a92be5b023b7155350f3d12d5b